### ISSUE TITLE
Unable to sync with Dropbox (401 unauthorized error)

### ISSUE THREAD

**Issue by lfdebrux** [benweet/stackedit#1782]
When using StackEdit with Dropbox, synchronization works temporarily when initially connected to the account, however after a while it fails with message 'HTTP error 401 on Dropbox location.'.

Disconnecting and then reconnecting the account fixes the problem, but only temporarily.

Looking in the dev console I can see the following messages being logged:

```
Failed to load resource: the server responded with a status of 401 (Unauthorized)
syncSvc.js:514 Objectbody: "{\"error_summary\": \"expired_access_token/\", \"error\": {\".tag\": \"expired_access_token\"}}"headers: {cache-control: 'no-cache', content-length: '85', content-type: 'application/json'}status: 401[[Prototype]]: Object
(anonymous) @ syncSvc.js:514
```

**Comment by joshunger** [comment]
I'm seeing the same issue 

<img width="698" alt="image" src="https://user-images.githubusercontent.com/2301847/164910365-4807f5f6-7858-4e49-9496-f1ab0d5b1ff2.png">

**Comment by exanrauzer** [comment]
Yes, I was hoping to use stackedit on mobile and Zettlr on laptop connected by dropbox. It is not possible.

**Comment by ekskog** [comment]
This is still around. I guess it has to do with the TTL of the dropbox token. Will it ever be fixed?

**Comment by alexgreen** [comment]
I've had this issue several times on OSX Chrome and found that clearing the site data and cache resolved it, at least temporarily:
1. Open stackedit.io in browser.
2. Click on the lock icon in browser URL bar and select "coockies and site data", then select "manage cookies and site data".
3. In the resulting pop-up press the trash icon next to stackedit.io.
4. Reload the page.
5. Relink your Google workspace (this might actually happen automatically upon refresh). All your stackedit directories and data should then be repopulated. 
6. Relink your publishing location. (Click "Add Dropbox account" or whatever you use) 
7. Click "Publish now" and it should work
